debezium-server/debezium-server-pubsub/src/main/java/io/debezium/server/pubsub/PubSubChangeConsumer.java

```diff
@@ -141,10 +157,20 @@ void connect() {
                     .setLimitExceededBehavior(FlowController.LimitExceededBehavior.Block)
                     .build());
         }
+        
+        if (address.isPresent()) {
```

Comment:
If default value is empty string then I belive this will be always executed.

Reply:
Good point; what does an empty value for the address represent though? Does having that default value even make sense? For databases, we usually assume `localhost:<standard port>` (e.g. `localhost:5432` for PG), but for such cloud-based service there seems to be no sensible default?

Reply:
I am not sure under which circumstances it would make sense to provide an address, other than when pointing towards an emulator running on the same host as debezium-server itself, which would be in a test scenario. 

Nevertheless, I have removed the `defaultValue` attribute on the `@ConfigValue` attribute.